packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1718,18 +1757,20 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   closeLocalStream() {
     return Media.stopTracks(this.mediaProperties.audioTrack)
```

Comment:
Is there a reason to not use `stopStream` here instead?

Reply:
Yes, I noticed the original code using `stopTracks`. This is due to the fact that local video, audio and share tracks are not stored in a stream in the `meeting.mediaProperties` instance. They are stored as individual tracks.